### PR TITLE
Test Azure-Samples/cargotracker-wls-aks periodically

### PR DESCRIPTION
**Pull request by galiacheng**
Test: https://github.com/azure-javaee/cargotracker-wls-aks/actions/runs/8430189814

### CODE REVIEW

File: .github/workflows/setupWlsAks.yml [hunk 1]
```diff
@@ -5,16 +5,22 @@ on:
...
 
 # All of the secrets in the env section are required. Run setup.sh to
 # populate the git repository running this workflow with the necessary secrets.
     
 env:
-  aksRepoUserName: oracle
-  aksRepoBranchName: 278a527cb7d1eb44a5a65ccc02b6837fe42967db
-  aksClusterConfigMapName: wlsd-wdt-config-map-${{ github.event.inputs.disambiguationSuffix }}
+  aksRepoUserName: ${{ secrets.AKS_REPO_USER_NAME }}
+  aksRepoBranchName: 2024-03-13-2-Q1
```
Comment (edburns) [inline .github/workflows/setupWlsAks.yml]: Update to `2024-03-21-1-Q1`.

File: README.md [hunk 2]
```diff
@@ -87,13 +87,13 @@ After the Maven command completes, the WAR file locates in `${DIR}/cargotracker/
...
-cp ${DIR}/cargotracker/.scripts/setup-env-variables-template.sh ${DIR}/cargotracker/.scripts/setup-env-variables.sh
+cp ${DIR}/cargotracker/src/test/aks/setup-env-variables-template.sh ${DIR}/cargotracker/src/test/aks/setup-env-variables.sh
 ```
 
-Open `${DIR}/cargotracker/.scripts/setup-env-variables.sh` and enter the following information. Make sure your Oracle SSO user name and password are correct.
+Open `${DIR}/cargotracker/src/test/aks/setup-env-variables.sh` and enter the following information. Make sure your Oracle SSO user name and password are correct.
 
 ```bash
-export WLS_AKS_REPO_REF="278a527cb7d1eb44a5a65ccc02b6837fe42967db" # oracle/weblogic-azure reference
+export WLS_AKS_REPO_REF="2024-03-13-2-Q1" # oracle/weblogic-azure reference
```
Comment (edburns) [inline README.md]: Update to tag `2024-03-21-1-Q1`.

File: README.md [hunk 3]
```diff
@@ -227,7 +214,7 @@ DB_CONNECTION_STRING="jdbc:postgresql://${DB_SERVER_NAME}.postgres.database.azur
 
 ### Prepare deployment parameters
 
-Several parameters are required to invoke the Bicep templates. Parameters and their value are listed in the table. Make sure the variables have correct value.
+In the following sections, you use solution templates in [oralce/weblogic-azure](https://github.com/oracle/weblogic-azure/tree/2024-03-13-2-Q1/weblogic-azure-aks/src/main/arm) to provision resources. Several parameters are required to invoke [mainTemplate.json](https://github.com/oracle/weblogic-azure/blob/2024-03-13-2-Q1/weblogic-azure-aks/src/main/arm/mainTemplate.json). Parameters and their value are listed in the table. Make sure the variables have correct value.
```
Comment (edburns) [inline README.md]: Update the tag with `2024-03-21-1-Q1`.

File: src/test/aks/setup-env-variables-template.sh [hunk 4]
```diff
@@ -1,9 +1,9 @@
-export WLS_AKS_REPO_REF="278a527cb7d1eb44a5a65ccc02b6837fe42967db" # oracle/weblogic-azure reference
-export RESOURCE_GROUP_NAME="abc1110rg" # customize this
-export STORAGE_ACCOUNT_NAME="stgwlsaks$(date +%s)" # storage account name
-export DB_SERVER_NAME="wlsdb$(date +%s)" # PostgreSQL server name
-export DB_PASSWORD="Secret123456" # PostgreSQL database password
-export MY_ORACLE_SSO_USER="user@contoso.com" # replace with your Oracle Account user name.
-export MY_ORACLE_SSO_PASSWORD="Secret123456" # replace with your Oracle Account password.
-export MY_WEBLOGIC_ADMIN_USER_NAME="weblogic" # weblogic admin user name
-export MY_WEBLOGIC_ADMIN_PASSWORD="Secret123456" # weblogic admin password
+export WLS_AKS_REPO_REF="2024-03-13-2-Q1" # oracle/weblogic-azure reference
```
Comment (edburns) [inline src/test/aks/setup-env-variables-template.sh]: Update the tag to `2024-03-21-1-Q1`.

**Review by edburns — CHANGES_REQUESTED** [review]
Update tag to `2024-03-21-1-Q1` as shown inline.

File: src/test/aks/setup-env-variables-template.sh (hunk 4 above)
Comment (galiacheng) [reply src/test/aks/setup-env-variables-template.sh]: Resolved in https://github.com/Azure-Samples/cargotracker-wls-aks/pull/7/commits/635c0079fb9d5c31d681d07e6cc147bb446393c1

File: README.md (hunk 3 above)
Comment (galiacheng) [reply README.md]: Resolved in https://github.com/Azure-Samples/cargotracker-wls-aks/pull/7/commits/635c0079fb9d5c31d681d07e6cc147bb446393c1

File: README.md (hunk 2 above)
Comment (galiacheng) [reply README.md]: Resolved in https://github.com/Azure-Samples/cargotracker-wls-aks/pull/7/commits/635c0079fb9d5c31d681d07e6cc147bb446393c1

File: .github/workflows/setupWlsAks.yml (hunk 1 above)
Comment (galiacheng) [reply .github/workflows/setupWlsAks.yml]: Resolved in https://github.com/Azure-Samples/cargotracker-wls-aks/pull/7/commits/635c0079fb9d5c31d681d07e6cc147bb446393c1

**Review by edburns — APPROVED** [review]

**Pull request merged**